docs/conferences/sessions.md

```diff
@@ -0,0 +1,87 @@
+This page explains the purpose of _Sessions_ in Indico and how to use them.
+
+Put simply, sessions are a way to group related contributions when creating a timetable.
+As an example, a conference could be split into a welcome session, a session dedicated to talks, workshop sessions etc.
+
+In this part, we will cover how to create and manage sessions from the organizers' point of view. If you want to learn how sessions are used when creating a timetable, see [this page](./timetable.md).
+
+### Creating a session type
+
+Before creating a session, we will first create a session type. Session types can be used to better organize your sessions. To create a new session type, navigate to the management area of an event and select _Sessions_ under _Organization_.
```

Comment:
I'd clarify that there's no need to create session types in most events unless you have poster sessions (for those it's very important because of the "poster" flag that prevents the timetable from trying to show individual contributions with start times in those sessions)

Reply:
Also, I see you use "Day 1" in the example. This is probably not a good one because this is now how session types should be used. Something like "Poster session" (and "Plenary talks" if you want more than one) would be a better fit.